### ISSUE TITLE
Improving Accessibility for Woo Commerce iOS

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-ios#6554]
- [ ] The button for displaying order details just shows button as the name when reading in Voice Over/displaying them in Voice Control
- [ ] Feedback Survey Close Button Is Only Displayed As `Button` when using Voice Control to Display Button Names.
- [ ] Lack of Dynamic Type for the My Store tab.
- [ ] No Dynamic Type when viewing the tags for a product.

Details for these findings are here: https://docs.google.com/document/d/1Fjzh2X0qgNkeKZR694YqRuq2ZHY522s27OcRGxW8CQk/edit?mode=html

**Comment by JB184351** [comment]
Closing this issue in favor of separate issues.

https://github.com/woocommerce/woocommerce-ios/issues/6595
https://github.com/woocommerce/woocommerce-ios/issues/6593
https://github.com/woocommerce/woocommerce-ios/issues/6596
https://github.com/woocommerce/woocommerce-ios/issues/6597
https://github.com/woocommerce/woocommerce-ios/issues/6598
